mappings/net/minecraft/client/crash/MinecraftCrash.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_776 net/minecraft/client/crash/MinecraftCrash
```

Comment:
Maybe CrashInfo? Minecraft in the name is a bit unnecessary as this is minecraft ;)